autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -60,7 +60,7 @@
     <let name="launch_dummy_vehicle" value="true" unless="$(var scenario_simulation)"/>
     <let name="launch_dummy_localization" value="true"/>
 
-    <include file="$(find-pkg-share simulator_launch)/launch/simulator.launch.xml">
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_simulator_component.launch.xml">
```

Comment:
There is also a digital twin simulation and a rosbag replay simulation, so the file name is better to clear that it is for a planning simulation.

Reply:
Thank you!
Do you mean something like `/launch/components/tier4_planning_simulator_component.launch.xml`?
Maybe we also should modify the name `tier4_simulator_launch` in autoware.universe?

Reply:
> Do you mean something like /launch/components/tier4_planning_simulator_component.launch.xml?
> Maybe we also should modify the name tier4_simulator_launch in autoware.universe?

Yes :+1:

Reply:
I see, thank you. 

But changing the name of `tier4_planning_simulator_component.launch.xml` and `tier4_simulator_launch` sounds out of scope of this PR.

How about we first go on with `tier4_simulator_component.launch.xml` for this PR, and discuss about renaming the launch files in another PR (or maybe internal / public discussion)?

Reply:
OK :+1: 
I'll resolve this conversation